frontend/src/components/on_boarding/OnBoardingHero.js

```diff
@@ -16,6 +16,7 @@ export default ({ utmSource }) => (
     </div>
     <div className="-requirements">
       You'll need a GitHub account, a repository with over 100 stars that you own & at least 2 contributors.
+      Please read our Terms of Service <a href="https://docs.google.com/document/d/1-hajYd7coL05z2LTCOKXTYzXqNp40kPuw0z66kEIY5Y/pub">here</a>
```

Comment:
"Terms of Service" linked to the doc is cleaner. No need for "Please read our" or "here".